exercises/anagram/description.md

```diff
@@ -1,4 +1,5 @@
-Given a word and a list of possible anagrams, select the correct sublist.
+Given a word and a list of candidates, select the sublist of anagrams of the given word.
+An anagram is a rearrangement of characters to form a new word.
```

Comment:
The definition should probably come first, rather than after.  It almost "sounds" like an afterthought, otherwise.

Reply:
I would use "letters" rather than "characters" as we are solving using letters, rather than characters.

Reply:
Thanks! I agree with the use of letters instead of characters. Also it is more clear with the definition first.